### PR TITLE
Add BC7 support

### PR DESCRIPTION
**Pull request by kg**
Depends on https://github.com/FNA-XNA/FNA3D/pull/121

### CODE REVIEW

File: src/Graphics/SurfaceFormat.cs [hunk 1]
```diff
@@ -108,5 +108,13 @@ public enum SurfaceFormat
 		/// Byte encoding is in sRGB colorspace, read in shader in linear colorspace.
 		/// </summary>
 		Dxt5SrgbEXT,
+		/// <summary>
+		/// BC7 block texture format
+		/// </summary>
+		BC7EXT,
```
Comment (flibitijibibo) [inline src/Graphics/SurfaceFormat.cs]: We have to be goofy and do "Bc7EXT", emphasis on 'c', to stick with the XNA casing scheme. (Also I think Supergiant capitalizes it this way in their C# engine.)

**Review by flibitijibibo — APPROVED** [review]
Wouldn't let me delete the comment, this is fine now

**Comment by kg** [comment]
Before we merge do I need to put a FNA3d bump in here? Or will you do it once we merge the FNA3d side?

**Comment by flibitijibibo** [comment]
I'll do it on my end, so nothing to worry about here.

**Pull request merged**